admin/src/utils/get-fields-layout.js

```diff
@@ -55,24 +56,25 @@ const normalizeField = ( field, schema ) => {
   // We don't want the `label` prop for the rendered input component.
   let input = omit( field.input, 'label' );
 
-  // Replace `label` prop in custom config with formatted object.
-  if ( label ) {
-    input.intlLabel = formatString( label );
+  // Ignore label field when intlLabel is already set (default fields)
+  if( !input.intlLabel ) {
+    // Replace `label` prop in custom config with formatted object.
+    input.intlLabel = formatString( name, 'label', label || name );
```

Comment:
@davidlueder Please use `??` instead of `||`. It's a better practice because it avoids certain issues with falsey values.